admin/nodes/views.py

```diff
@@ -467,3 +470,19 @@ def post(self, request, *args, **kwargs):
                                     ' if the problem persists get a developer to fix it.')
 
         return redirect(reverse_node(self.kwargs.get('guid')))
+
+
+class RemoveStuckRegistrationsView(StuckRegistrationsView):
+    template_name = 'nodes/remove_registrations_modal.html'
+
+    def post(self, request, *args, **kwargs):
+        from osf.management.commands.force_archive import verify
```

Comment:
@Johnetordoff Is there a reason to import in the view instead of the top of the file?

Reply:
it creates a strange circular import where the admin app hangs on start up and never fully loads

Reply:
Hm, OK. I'll move these back and add a comment.